docs/source/usage/parameter_scans.rst

```diff
@@ -12,11 +12,13 @@ Each workflow consists of a ``Snakefile`` in which the workflow is defined using
 How to use
 """"""""""
 
-The example presented here performs several simulations with given input parameters.
+The example presented here performs several simulations with given input parameters with the LaserWakefield example of PIConGPU on hemera.
+
+#. Copy the ``Snakefile`` and ``config.yaml`` which are located in the PIConGPU source code here: ``picongpu/share/picongpu/examples/LaserWakefield/lib/python/snakemake/``. There also a ``requirements.txt`` and some exemplary parameters ``params.csv`` can be found which are compatible with the LaserWakefield example.
 
 #. Make shure you have ``snakemake`` and the ``snakemake-executor-plugin-slurm`` installed and activated.
 
-#. Adjust the profile ``config.yaml``:
+#. Adjust the profile ``config.yaml`` :
```

Comment:
There is no reason to add a space, no?

Reply:
True, this was probably by accident. This bullet point is rendered in bold letters in readthedocs and I thought I could somehow change this, but apparently this is some sort of convention for bullet points with sub-bullet points.